.github/workflows/check-internal-deps.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       matrix:
         go-version:
-          - 1.19
+          - 1.21
```

Comment:
do a repo wide search for `1.19`. There's a lot more places

Reply:
let's do this in a separate PR, since `go.mod` isn't being updated to use `1.21` yet